index.js

```diff
@@ -1,9 +1,16 @@
 const express = require('express');
+const axios = require('axios');
 const app = express();
 
+const SUPER_SECRET_PASSWORD = 'lovethebomb69';
```

Comment:
Dude, you can't put the password in the file.
Remove this in the next commit

You need to learn much more before making better boombas